setup.py

```diff
@@ -48,7 +48,7 @@
     "pytest-cov>=2.8.0,<3.0",
     "pytest>=4.6.0,<5.0.0",
     "reana-commons[kubernetes]>=0.7.0.dev20200203",
```

Comment:
Do we need to pin also r-commons to less than 0.8? Would make sense since this is going to the `maint-0.7` branch only... *(untested)*

Reply:
True, didn't notice as I'm working on a r-commons' maint-0.7-based branch directly. I pushed the change, maybe @ParthS007 can test it in his PR?

Reply:
Tested it already myself https://github.com/reanahub/reana-workflow-engine-yadage/pull/164#issuecomment-708279373